.gitignore

```diff
@@ -16,4 +16,13 @@
 .idea
 
 # Auxiliary files generated by some antlr tools
-.antlr
\ No newline at end of file
+.antlr
+
+# VScode debug files
+*/launch.json
+
+# MacOS autogenerated files
+*/.DS_Store
```

Comment:
Hier könntest du einfach nur `.DS_Store` schreiben, dann werden nicht nur Dateien in Unterverzeichnissen ignoriert.

Reply:
übernommen